.github/workflows/ci.yml

```diff
@@ -43,4 +68,36 @@ jobs:
 
     - name: Test
       run: |
-        cabal test all --enable-tests
+        cabal test --enable-tests --test-show-details=direct
```

Comment:
Would it be a good idea to add one more step here, like, `Documentation` and run `cabal haddock`?